test/correctness/simd_op_check.h

```diff
@@ -94,27 +94,32 @@ class SimdOpCheckTest {
         std::string asm_filename = output_directory + "check_" + name + ".s";
         f.compile_to_assembly(asm_filename, arg_types, target);
 
-        std::ifstream asm_file;
-        asm_file.open(asm_filename);
-
         bool found_it = false;
 
         std::ostringstream msg;
         msg << op << " did not generate for target=" << target.to_string() << " vector_width=" << vector_width << ". Instead we got:\n";
 
-        std::string line;
-        while (getline(asm_file, line)) {
-            msg << line << "\n";
+        {
+            std::ifstream asm_file;
```

Comment:
What actually changed here?

Reply:
code restructuring that no longer matters. Will revert.